### PR TITLE
chore: dependency and lint bump

### PR DESCRIPTION
**Pull request by faddat**
Closes #<issue number>. _in case of a bug fix, this should point to a bug or any other related issue(s)_

### What does this PR do?

* ensures that we don't import versions below sdk v0.46.0

### How to test?

* make a 46 chain with ignite or try to upgrade a 45 chain with ignite
 
## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Include tests (N/A)
- [ ] Respect code style and lint (hope so)
- [ ] Update documentation (*.md) (if needed) (should not be needed)

### CODE REVIEW

**Comment by aljo242** [comment]
Hey @faddat overall looks good to me.  Can we not include the `.gitpod` file in this PR?

**Review by aljo242 — APPROVED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - package-ecosystem: gomod
+    directory: "/"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 20
+  - package-ecosystem: npm
+    directory: "/"
+    schedule:
+      interval: daily
+    open-pull-requests-limit: 20
```
Comment (lubtd) [inline .github/dependabot.yml:17]: I don't think npm is needed

**Review by lubtd — COMMENTED** [review]
Thanks for the PR @faddat !

**Comment by aljo242** [comment]
closing due to lack of activity

**Pull request closed without merging**